pkg/cloud-controller-manager/loadBalancer.go

```diff
@@ -70,18 +67,8 @@ func (l *LoadBalancerManager) GetLoadBalancer(ctx context.Context, clusterName s
 }
 
 func (l *LoadBalancerManager) GetLoadBalancerName(ctx context.Context, clusterName string, service *v1.Service) string {
-	// If users don't specify cluster name by the command flag, we adopt the cluster name from the cloud config file
-	if clusterName == "" || clusterName == "kubernetes" {
-		clusterName = l.clusterName
-	}
-	name := clusterName + "-" + service.Namespace + "-" + service.Name
-	// https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names
-	// The name contains no more than 253 characters
-	if len(name) > 253 {
-		name = name[:253]
-	}
-
-	return name
+	// refer to https://kubernetes.io/docs/concepts/overview/working-with-objects/names
+	return "a" + string(service.UID) + "a"
```

Comment:
The suffix "a" seems unnecessary.
Can we use a prefix like "lb-" or "svc-" which looks less hacky